src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/RotatingStar.cpp

```diff
@@ -614,9 +691,10 @@ RotatingStar::variables(
   using std::max;
   return {hydro::relativistic_specific_enthalpy(
       Scalar<DataType>{DataType{max(1.0e-300, get(rest_mass_density))}},
-      equation_of_state_.specific_internal_energy_from_density(
-          rest_mass_density),
-      equation_of_state_.pressure_from_density(rest_mass_density))};
+      equation_of_state_->specific_internal_energy_from_density(
+          Scalar<DataType>{DataType{max(1.0e-300, get(rest_mass_density))}}),
+      equation_of_state_->pressure_from_density(
```

Comment:
I think the cleaner approach would be to retrieve the pressure using the `get(get<hydro::Tags::Pressure<DataType>>(variables(vars, x, tmpl::list<hydro::Tags::Pressure<DataType>>{})))`, the reason being that then you only need the number `1e-300` for the pressure once there.